src/test/ui/parser/removed-syntax-field-let.stderr

```diff
@@ -1,10 +1,15 @@
 error: expected identifier, found keyword `let`
   --> $DIR/removed-syntax-field-let.rs:2:5
    |
-LL | struct S {
-   |        - while parsing this struct
 LL |     let foo: (),
    |     ^^^ expected identifier, found keyword
+   |
+   = help: see <https://doc.rust-lang.org/book/ch05-01-defining-structs.html> for more information
+help: remove the let, the `let` keyword is not allowed in struct field definitions
+   |
+LL -     let foo: (),
+LL +      foo: (),
```

Comment:
Is there one additional space character here in the generated help message (which should not be there) or am I missing something?

Reply:
You are removing `"let"` from `"let foo: (),"`, so you get `" foo: (),"`. Replacing `self.prev_token.span` with `self.prev_token.span.until(self.token.span)` or something alike should help.